src/locales/messages.ts

```diff
@@ -1698,18 +1889,18 @@ export default defineMessages({
     id: 'MonthOverMonthChange',
   },
   Name: {
-    defaultMessage: 'EN Name',
-    description: 'Name',
-    id: 'CostModelsWizardSourceTableName',
+    defaultMessage: '{count, plural, one {EN name} other {EN names}}',
+    description: 'Name plural or singular',
+    id: 'Name',
```

Comment:
We're using Name in a few places. 

If we're going to make this change, need to correct the other usages. It defaults to "names", which is not how it was intended to be used.

I see you changed two components below, but it's also used in:

dataToolbar.tsx (messages.Name)
addSourceWizard (messages.Name)
awsDetails/detailsTable (messages.Names)
addSourceStep (messages.Name)
explorerTable (messages.Names)
sourcesTable (messages.Name)
costModel/table (messages.Name)

Reply:
If the default is "names", then perhaps the message ID should also be Names?

Reply:
OK for now until we discuss it I have changes those missed usages with a "count"